### PR TITLE
Script to create alert app packs

### PR DESCRIPTION
**Pull request by cstavitsky**
- Script to allow creation of preconfigured alerts (with no actions set) for customers
- See README for instructions/details

### CODE REVIEW

File: app_packs/alerts/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+## Summary:
+
+#### The essentials
+- Run this API script to create Alert App Pack rules for projects within an org.
+- :warning: By default, it will create alerts for every project in an org.
+- You can also pass a list of projects to create alerts for (see Run Script section).
+  
+#### Other things to know
+- The alerts are created without being assigned any teams. SEs or customers will need to assign teams.
+- The alerts are created with zero actions. SEs will need to encourage customers to add actions (i.e. set up an email/slack notification)
```
Comment (sentryadam0000345) [inline app_packs/alerts/README.md:12]: Maybe just make reference to the section you outlined below


- The alerts are created with zero actions. SEs will need to encourage customers to add actions (i.e. set up an email/slack notification)





[3:41](https://sentry.slack.com/archives/D02AMS00GH0/p1667428904696209)
I would change this to add actions and further personalize, SEs should use this as a point of engagement with customers
[3:41](https://sentry.slack.com/archives/D02AMS00GH0/p1667428907322439)
something like that
[3:41](https://sentry.slack.com/archives/D02AMS00GH0/p1667428917944749)
basically rehashing the Notion stuff you added but just a bit

File: app_packs/alerts/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+This script creates alerts with:
+- no specified action
+- certain default values that can and should change based on customer needs
+
+You should walk the customer through the alerts you've created, on a call. Explain that they need to enable an action for any alerts they're interested in, and point out a few values that have been configured with defaults, that they can change as desired.
+
+For example:
+
+1. For the "Regression Error Occurred" alerts, we alert on the latest release. This is a fine default, but let customers know they could change it to remove that filter and alert on ANY regression if they wanted to.
```
Comment (sentryadam0000345) [inline app_packs/alerts/README.md:74]: This is a good opportunity to ask a Discovery question such as, "If this alert is not helpful for you, what alert condition would be, what kind of alerts have been helpful in the past, etc"
